pkg/kubelet/cm/cpumanager/topology/topology.go

```diff
@@ -236,8 +236,7 @@ func Discover(machineInfo *cadvisorapi.MachineInfo) (*CPUTopology, error) {
 					}
 				}
 			} else {
-				klog.Errorf("could not get unique coreID for socket: %d core %d threads: %v",
-					core.SocketID, core.Id, core.Threads)
+				klog.ErrorS(nil, "Could not get unique coreID for socket", "socket", core.SocketID, "core", core.Id, "threads", core.Threads)
```

Comment:
```suggestion
				klog.ErrorS(nil, "Could not get unique coreID for socket", "socketID", core.SocketID, "coreID", core.Id, "threads", core.Threads)
```

Reply:
I don't think this is consistent, we should leave this as is.